Gemfile

```diff
@@ -0,0 +1,6 @@
+source "https://rubygems.org"
+
+group :test do
+  gem 'rspec'
```

Comment:
Agree logstash should not be take into account. Rspec3 should work good here.

If you ask me, I would to block to the 3.x branch.